### PR TITLE
Create Preprocessing Documentation for Labels (np_utils.to_categorical)

### PR DESCRIPTION
**Pull request by RamaneekGill**
Includes useful documentation that was only found through an example in the sequential tutorial.

The use of `keras.utils.np_utils.to_categorical(labels, nb_classes` is hard to find in current documentation and is very useful for many categorical classification tasks that many students/beginners start out with.

`to_categorical` belongs to the `np_utils` file and not a preprocessing one so this addition could be misleading in terms of code structure but I believe it is useful to include in the current documentation.

Link to method documented: https://github.com/fchollet/keras/blob/master/keras/utils/np_utils.py

To test follow documentation: https://github.com/fchollet/keras/blob/master/docs/README.md

### CODE REVIEW

**Comment by fchollet** [comment]
I believe this should be a note inserted in the `objectives` page: "`categorical_crossentropy` expects categorical binary labels, here's a way to convert your integer labels to categorical".

The documentation follows the structure of the codebase and as such this page wouldn't fit.

**Comment by fchollet** [comment]
I have updated the docs to reflect this.
